rdrf/rdrf/templates/dashboard/widget/consents.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'dashboard/widget/_widget_base.html' %}
+
+{% load i18n %}
+
+  {% block card_header %}
+    {% if not widget.title %}
+      {% trans "Consents" %}
+    {% else %}
+      {{ block.super }}
+    {% endif %}
+
+    <span class="badge pull-right {% if consent_status.valid %}bg-success{% else %}bg-danger{% endif %}">
+      {% if consent_status.valid %}
+        {% trans "Complete" %}
+      {% else %}
+        {% trans "Incomplete" %}
+      {% endif %}
+    </span>
+  {% endblock %}
+
+  {% block card_body_text %}
+    <p>{% trans "You have approved" %} {{ consent_status.completed }} {% trans "out of" %} {{ consent_status.total }} {% trans "consents" %}.</p>
```

Comment:
Should use `blocktrans` with placeholders. Also, we call them 'consents' but I'm not sure if that term is understood by the broader population. Maybe 'consent questions'?

Reply:
The term 'Consents' is also used in the registry to identify the Consents page in the sidebar menu and also as a heading for the consent page. So it might be ok to leave it as is?

Reply:
👍 ok